### DISCUSSION TITLE
Custom workspace icons

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#866] [Ideas]
### What feature would you like to see?

Coming over from Firefox with a Sidebery set-up and I will say, it's great. I feel it's helping me stop tab hoarding which Sidebery didn't really do so, thanks for that. :+1: 

One thing I would like is for the ability to add custom icons for the workspace. On Sidebery, I can choose one of their own icons and colours, as well as custom text/colours or an image. Simple Tab Groups gives you colour-changeable symbols or an image. On Vivaldi, I can choose any emoji. It'd be nice to have the ability to use a specified emoji or icon for the workspace, as I feel the emojis I have now are limited.

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/279_

**Comment by ChristinWhite** [comment]
Please support more than just emoji, uniform icons are a lot less distracting, at least to me. Being able to use a custom SVG Sidebery or SigmaOS style is a godsend in those browsers.

**Comment by staffanmowitz** [comment]
I believe the best option would be to integrate a good open source icon library such as [Lucide](https://lucide.dev/)

**Comment by kristijanribaric** [comment]
You can already assign icons to a workspace.
To add more icons to the selection go to the `about:config` page and search for `zen.workspaces.icons` preference. Update it with icons you want to be able to select for the workspace.

**Reply by blakegearin** [reply]
This is somewhat useful to add emojis that are missing but also sort of misses the mark. Emojis and icons are very different concepts technically. Arc uses [ionicons](https://ionic.io/ionicons), and with an emoji-based system there's no way to bring your own icons.

**Reply by ChristinWhite** [reply]
Agree 100% with this, emoji are non-uniform and look messy but more importantly they aren't designed for readability at smaller sizes like being used as tiny workspace icons. Ionicons or [Lucide](https://lucide.dev/) would be much better, but the best option would be to also allow us to use custom SVGs. This is the approach both [Sidebery](https://github.com/mbnuqw/sidebery) and [SigmaOS](https://sigmaos.com/) have taken, Emoji + an icon set + custom SVGs.

**Reply by DataHearth** [reply]
That would be a huge UX advantage. I've been looking for a way to either find the icon library or to inject SVGs to customize it. 
+1 !

**Reply by Zeta611** [reply]
`zen.workspaces.icons` option is gone now.

**Comment by filintodelgado** [comment]
This was posted in August and still there seams to be no plans for this feature, it is the only thing holding me to switch from Firefox.

**Comment by Zeta611** [comment]
Custom icons and/or emojis would be awesome! I've been migrating from Arc and some emojis are missing to replicate the workspaces I had there. And `zen.workspaces.icons` option has been removed.

**Comment by levifig** [comment]
Why has this issue been closed as resolved? It'd be perfectly fine to close it as rejected or duplicate, but definitely not "resolved"… right? 😅

**Comment by Jarvvski** [comment]
Would love this feature as a recent migrator from Arc
